src/index.html

```diff
@@ -2,20 +2,69 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <p
+        class="card__product-name"
+      >
+         APPLE A1419 iMac 27" Retina <br>
+         5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p
+        class="card__product-code"
+      >
+         Product code: 195434
+      </p>
+      <div class="card__stars-wrapper">
+        <div class="stars stars--4 card__stars-block">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p
+          class="card__stars-review"
+        >
+           Reviews: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-syntax">Price:</p>
+        <p class="card__price-value">$2,199</p>
+      </div>
+      <a
+      class="card__purchase-button "
+      href="#"
+      data-qa="hover"
+      >
```

Comment:
Fix the code style here 
```suggestion
      <a
        class="card__purchase-button "
        href="#"
        data-qa="hover"
      >
```